pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.hsl</groupId>
     <artifactId>transitdata-common</artifactId>
-    <version>0.7.6</version>
+    <version>0.7.7</version>
```

Comment:
since we haven't released 7.6 we don't need to bump this up..?

Reply:
Oh yes, so we bump the version only when merging to master?

Reply:
I think it would make sense, it seems strange if there's some version labels missing in between..? 

And I'd keep the "release-threshold" quite low, release small and often.

Reply:
Sounds good.